validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -117,8 +118,16 @@ func (c *grpcValidatorClient) WaitForActivation(ctx context.Context, in *ethpb.V
 }
 
 // Deprecated: Do not use.
```

Comment:
Do you need to remove this?

Reply:
I added this comment because `WaitForChainStart` is technically deprecated (according to `validator.pb.go`), but it's still used throughout the codebase.